Mage/src/main/java/mage/target/targetadjustment/TargetAdjuster.java

```diff
@@ -12,6 +12,7 @@
 public interface TargetAdjuster extends Serializable {
 
     // Warning: This is not Copyable, do not use changeable data inside (only use static objects like Filter)
+    // Note: in playability check for cards, targets are not adjusted.
```

Comment:
Original commit 9c52dfa78516ae32b8df6b634bc3425cd3456b0f  reverted from master due wrong usage. Current PR don’t use that code, but it have docs about skip adjusting.

1. Is it fixed a bugs in current master code or just a refactored for better class usage?
2. Is it require additional fixes due not working adjusters in playable check?

Reply:
1) This fixes the same major bug as 9c52dfa, where cards that should be playable are not, such as [[Long River's Pull]], where the base spell's target is more restrictive than the adjusted version.
2) There are some cases where a spell that isn't currently playable is highlighted as if it were, such as [[Bloodchief's Thirst]] if you don't have enough mana to kick it and the only creatures on the battlefield have a higher MV, but I don't think there's currently any reasonable way to do that.